src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -136,6 +136,7 @@ export type Preferences = {
     selectedChainId: ChainId | 'all'
   }
   hasSeenRatingModal: boolean
+  walletIdToDefaultTcyAccountId: Record<WalletId, AccountId>
```

Comment:
Not sure this is the right home for that!

Preference slice shouldn't be related to some business feature scoped logic, the only purpose of this is the TCY feature, it would be a good use of zustand or its own scoped persisted context/slice

We could maybe use this new slice to select the selected account directly and avoid props drilling at the same time, wdyt?